packages/common/src/initiatives/_internal/InitiativeUiSchemaEdit.ts

```diff
@@ -142,6 +143,47 @@ export const buildUiSchema = async (
           },
         ],
       },
+      {
+        type: "Section",
+        labelKey: "{{i18nScope}}.sections.featuredContent.label",
```

Comment:
Now that this is a function, we should use template literals to inject the `i18nScope` directly: 
```ts
labelKey: ${i18nScope}.sections.featuredContent.label
```

same for all of the below